content/docs/get-started-with-neon/query-with-psql-editor.md

```diff
@@ -1,20 +1,124 @@
 
-In the console go to the Project Dashboard, click “Generate Token” button, and follow the instructions to save the password into .pgpass file
+## Connecting with Neon's psql quick auth
 
-Copy the connection string and run it in the shell:
+Neon's `psql` quick auth feature helps you quickly and conveniently authenticate a connection to a Neon project.
+
+1. In your terminal, run the following command:
+
+```bash
+psql -h pg.neon.tech
+```
+
+A response similar to the following is displayed:
+
+```bash
+NOTICE:  Welcome to Neon!
+Authenticate by visiting:
+    https://console.neon.tech/psql_session/6d32af5ef8215b62
+```
+
+2. In your browser, navigate to the provided link where you are asked to select an existing project or create a new project.
+
+- Selecting an existing project authenticates your connection to the selected project.
+
+- Selecting **Create new project** directs you to a **Project creation** page where you create a new project to connect to.
+
+After making your selection, you are directed to check the terminal. In the terminal, you are connected to the project, and  connection information similar to the following is displayed:
+
+```bash
+NOTICE:  Connecting to database.
+psql (14.5 (Ubuntu 14.5-0ubuntu0.22.04.1))
+Type "help" for help.
+
+user1=>
+```
+
+**_Note_**: When using _`psql` quick auth_ to connect, the `psql` prompt shows your local terminal user name instead of the database name that is shown for the other `psql` connection methods described in this topic. However, you are logged in to the default `main` database as the Neon `web_access` user, which you can verify by running this query:
+
+```bash
+user1=> SELECT current_user;
+ current_user 
+--------------
+ web_access
+(1 row)
+
+user1=> SELECT current_database();
+ current_database 
+------------------
+ main
+ (1 row)
+```
+
+## Connecting with an exported password
+
+**_Warning_**: Some operating systems allow non-root users to view process environment variables when using the `ps` command. For security reasons, consider using a password file in such cases. 
+
+To connect with an exported password:
+
+1. In your terminal, export the database user's password to the `PGPASSWORD` environment variable; for example:
+
+```bash
+export PGPASSWORD=En5v0dJoVpRL
+```
+
+The database user's password was provided to you when you created the project.
+
+2. Connect with the following command:
+
+```bash
+psql postgres://<user>:$PGPASSWORD@<project_ID>.cloud.neon.tech:5432/main
+```
+
+where:
+
+- `<user>` is the database user, which can be found on the Neon console **Dashboard** tab, under **Connection Details**. 
+- `<project_ID>` is the Neon project ID, which can be found on the Neon console **Settings** tab, under **General Settings**.
```

Comment:
IMO `project_ID` with capital id looks a bit inconsistent

Reply:
Fixed. Thank you for that!